vmprof/reader.py

```diff
@@ -43,12 +71,12 @@ class JittedCode(int):
     pass
 
 def wrap_kind(kind, pc):
-    if kind == VMPROF_ASSEMBLER_TAG:
-        return AssemblerCode(pc)
-    elif kind == VMPROF_JITTED_TAG:
-        return JittedCode(pc)
-    assert kind == VMPROF_CODE_TAG
-    return pc
+    cls = {
+        VMPROF_ASSEMBLER_TAG: AssemblerCode,
+        VMPROF_JITTED_TAG: JittedCode,
+        VMPROF_CODE_TAG: lambda x: x,
+    }[kind]
```

Comment:
if it doesn't matter than please don't change it, ifs are far more readable IMO :-) As for the measuring: timeit is terrible, don't use it, please use something that at the very least does average as opposed to min. As far as cpython times go, I don't care, there is never a reason to use cpython for parsing files. Any chance you can reformat the pull request in that spirit? Cheers, fijal